dclab/rtdc_dataset/plugins/plugin_feature.py

```diff
@@ -0,0 +1,66 @@
+"""
+.. versionadded:: 0.34.0
+"""
+
+import pathlib
+import importlib
+import sys
+
+from ..ancillaries import AncillaryFeature
+
+
+def load_plugin_feature(plugin_path):
+    """find an instanciate a PlugInFeature from a user-defined script"""
+    info = find_plugin_feature_script(plugin_path)
+    plugin_list = []
+    for ii in range(len(info["feature names"])):
+        ancill_info = {
+            "feature_name": info["feature names"][ii],
+            "method": info["method"],
+            "req_config": info["config required"],
+            "req_features": info["features required"],
+            "req_func": info["method check required"],
+            "priority": info["priority"],
+        }
+        plugin_list.append(PlugInFeature(info, **ancill_info))
+        # add feature label etc
+    return plugin_list
+
+
+def find_plugin_feature_script(plugin_path):
+    # find script, return info dict
+    path = pathlib.Path(plugin_path)
+    # insert the plugin directory to sys.path so we can import it
+    try:
+        plugin = importlib.import_module(path.stem)
+    except ModuleNotFoundError:
+        sys.path.insert(-1, str(path.parent))
+        plugin = importlib.import_module(path.stem)
+        # undo our path insertion
+        sys.path.pop(0)
+
+    return plugin.info
+
+
+def remove_plugin_feature(plugin_instance):
+    """Convenience function for removing a plugin instance"""
+    if isinstance(plugin_instance, PlugInFeature):
+        PlugInFeature.features.remove(plugin_instance)
+        PlugInFeature.feature_names.remove(plugin_instance.feature_name)
```

Comment:
Please add en `else` clause that raises an error. This function should not silently fail.